alacritty/src/display/window.rs

```diff
@@ -167,7 +168,7 @@ impl Window {
             .with_title(&identity.title)
             .with_theme(config.window.theme())
             .with_visible(false)
-            .with_transparent(true)
+            .with_transparent(config.window_opacity() < 1.)
```

Comment:
I believe we've had it this way in the past and that changed. Are you sure this is safe on all platforms? Because I'm not sure that this doesn't change Alacritty's behavior.

Reply:
You're right that the behaviour here is different on X11 and Windows, didn't consider other platforms when making this. Have reverted the change.